downstream/modules/configuring_cost_mgmt-operator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// assembly_adding_ocp_sources.adoc
+[id="configuring_cost_mgmt-operator"]
+[[configuring_cost_mgmt-operator]]
+== Installing and configuring the Cost Management Operator
+
+The Cost Management Operator collects the data required for cost management.    
+
+Begin adding your OpenShift Container Platform cluster as a source to cost management, then install and configure the Cost Management Operator from OperatorHub.
+
+[NOTE]
+====
+See https://docs.openshift.com/container-platform/4.3/operators/olm-understanding-operatorhub.html[Understanding the OperatorHub] in the OpenShift documentation for more information about OperatorHub.
+====
+
+.Prerequisites
+
+* OpenShift Container Platform 4.3 or newer
+
+.Procedure
+
+. In the cloud.redhat.com platform, open the *Sources* menu (https://cloud.redhat.com/beta/settings/sources/) to begin adding an AWS source to cost management:
+.. Navigate to *Sources* and click *Add a source* to open the Sources wizard.
+.. Enter a name for your source and click *Next*.
+.. Select *Cost Management* as the application and *OpenShift Container Platform* as the source type. Click *Next*.
+. In OpenShift, install and configure the Cost Management Operator from OperatorHub on your cluster, by searching for _cost management_.
+//Add link to instructions.
```

Comment:
Pretty sure this is the operatorhub.io link:
https://operatorhub.io/operator/cost-mgmt-operator/

I'm trying to chase down right now why it's not available on the website.

Reply:
These links look more official for downstream -- thanks for that! And for checking up on the .io instructions.
Is OperatorHub.io the upstream project, or is there a way to link to something from within the OCP cluster UI? (I was rewatching the demo and it didn't look possible).

Reply:
No way to link something from within the OCP cluster UI (no static link, since the customers cluster UI is variable).

OperatorHub.io is like a catalog from which customers can install operators from various providers: https://www.redhat.com/en/blog/introducing-operatorhubio-place-finding-kubernetes-native-services

Reply:
@dayleparker we've chased down the operatorhub.io item and the downside 🌧️ is I don't think we are going to have a webpage to reference. They only show up on the site if we make our operator work on Kubernetes (which it likely does) but doesn't provide value to an upstream Kubernetes user (since it sends data to cloud.redhat.com and things like the pull-secret won't exist).

So the best thing we can do is either direct follow the instructions they find in OpenShift (vaguely) or add them to our docs:
https://github.com/operator-framework/community-operators/blob/master/community-operators/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml#L101-L219
plus what Ashley pulled together: https://gist.github.com/abaiken/c69c6992f25ce7983a8631d53d0a7b0b#file-operator_installation-md

Reply:
Thanks for investigating, Chris! I chatted with Ashley about this too, and the plan is to add these instructions to the Cost docs (in the 'adding an OCP source' part), so people have them if needed, and they're easily searchable, but also mention that they can follow the instructions in OperatorHub, as you mention.

Ashley also said we should add:
- that the customer can only install the operator in the metering namespace

We will still have the "Learn more" link pointing to the 'add an OCP source' docs from the sources UI wizard, so I think that should close the loop.

Only remaining question now is - should we also provide steps to configure the operator via the OCP UI, as we saw in our demo meeting? We can tackle this question later too...

Reply:
I'd stick with CLI since that is what Metering does today. UI's change so often, probably safer also.